client/src/test/java/io/pravega/client/stream/ReaderGroupConfigTest.java

```diff
@@ -28,14 +28,21 @@
     public void testValidConfigWithScopedStreamName() {
         ReaderGroupConfig cfg = ReaderGroupConfig.builder()
                 .disableAutomaticCheckpoints()
-                .stream("scope/s1", getStreamCut("s1"))
-                .stream("scope/s2", getStreamCut("s2"))
+                .stream("scope", "s1", getStreamCut("s1"))
+                .stream("scope", "s2", getStreamCut("s2"))
+                .stream("s3", getStreamCut("s3"))
```

Comment:
One quick clarification: aren't we supposed to pass a default scope somewhere in this configuration?

Reply:
The default scope is passed while creating a `ReaderGroupManager` instance.
`io.pravega.client.admin.ReaderGroupManager#withScope(java.lang.String, java.net.URI)`